frontend/packages/dev-console/src/components/progressive-list/ProgressiveListFooter.tsx

```diff
@@ -22,7 +24,7 @@ const ProgressiveListFooter: React.FC<ProgressiveListFooterProps> = ({
         let preText = ' ';
         let postText = '';
         if (items.length - 1 === index) {
-          preText = items.length !== 1 ? ' and ' : ' ';
+          preText = items.length !== 1 ? t('devconsole~ and ') : ' ';
```

Comment:
We should make this one message for translation instead of concatenating string.

Reply:
@spadgett the first half of the text is being passed from the AdvancedSection component and the conjunction is being handled here in this component. Should we still try and make this as one single text for translation?

Reply:
> @spadgett the first half of the text is being passed from the AdvancedSection component and the conjunction is being handled here in this component. Should we still try and make this as one single text for translation?

I think we need to, otherwise the translators do not have enough context and the translated string might not make sense. I realize it's not trivial in this case.